### PR TITLE
Preliminary check in for legends

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

**Comment by jiwalker-usgs** [comment]
There is going to be a merge conflict shortly.  Also you checked in a bunch of documentation with windows newlines and such.

**Comment by thongsav-usgs** [comment]
@ldecicco-USGS how do you deal with the newlines issue on your setup?

**Comment by thongsav-usgs** [comment]
This PR should be cleaned up now.

File: man/MaumeeDV.Rd [hunk 1]
```diff
@@ -1,4 +1,4 @@
-% Generated by roxygen2 (4.1.1): do not edit by hand
+% Generated by roxygen2 (4.1.0): do not edit by hand
```
Comment (jiwalker-usgs) [inline man/MaumeeDV.Rd]: Big changes here.

**Pull request merged**